beaconChain/helpers/stateTransitionHelpers.ts

```diff
@@ -401,11 +408,22 @@ export function getBeaconProposerIndex(state: BeaconState, slot: int): int {
  * @param {int} value
  * @returns {boolean}
  */
-export function isPowerOfTwo(value: int): boolean {
-  if (value < 0) { throw new Error("Value is negative!"); }
-  if (value >= 2 ** 53) { throw new Error("Values greater or equal to 2 ** 53 are not supported yet!"); }
-  // TODO: Add support for values that are greater or equal to 2 ** 53
-  return value === 0 ? false : 2 ** Math.floor(Math.log2(value)) === value;
+export function isPowerOfTwo(value: BN): boolean {
+  if (value.ltn(0)) {
```

Comment:
I can't remember if the BN.js comparison operators require BN's as input, but if so all of these comparisons need to take BN instances instead of numbers

Reply:
luckily they have "+ n" methods for comparisons too